### PR TITLE
[UXE-3951] test: create countries and ip/cidr network lists

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
Add test E2E to network lists creation page
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [ ] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: cypress/e2e/network-lists.cy.js [hunk 1]
```diff
@@ -34,10 +34,194 @@ describe('Network Lists spec', () => {
...
 
+  it('should create a Countries Network List', function () {
+    // Act
+    cy.get(selectors.networkLists.createButton).click()
+
+    cy.get(selectors.networkLists.nameInput).clear()
+    cy.get(selectors.networkLists.nameInput).type(`${networkListName}`)
+
+    cy.get(selectors.networkLists.typeDropdown).click()
+    cy.get(selectors.networkLists.typeDropdown).find('li').eq(1).should('have.text', 'Countries').click()
```
Comment (william-tome) [inline cypress/e2e/network-lists.cy.js:45]: Confere na pasta de seletores como está sendo feito nesses cenários onde o valor é dinâmico.
Exemplo: edge dns

File: cypress/e2e/network-lists.cy.js [hunk 2]
```diff
@@ -34,10 +34,194 @@ describe('Network Lists spec', () => {
...
+
+    cy.get(selectors.networkLists.countriesMultiselect).click()
+    cy.get('.p-multiselect-item').eq(0).click()
+    cy.get('.p-multiselect-item').eq(1).click()
```
Comment (william-tome) [inline cypress/e2e/network-lists.cy.js:49]: O mesmo aqui e para os cenários seguintes

**Review by william-tome — COMMENTED** [review]
Pequenos ajustes nos seletores. De resto está muito bom.

File: cypress/e2e/network-lists.cy.js [hunk 3]
```diff
@@ -34,10 +34,194 @@ describe('Network Lists spec', () => {
...
+    cy.get(selectors.networkLists.saveButton).click()
+    cy.verifyToast('success', 'Your network list has been edited')
+    
+    //Assert
+    cy.get(selectors.networkLists.searchInput).clear()
+    cy.get(selectors.networkLists.searchInput).type(`${networkListName}{enter}`)
+    cy.get(selectors.networkLists.nameRow).should('have.text', `${networkListName}`)
+  })
+
+  it.only('should edit an IP/CIDR Network List', function () {
```
Comment (william-tome) [inline cypress/e2e/network-lists.cy.js:176]:
```suggestion
  it('should edit an IP/CIDR Network List', function () {
```

**Review by william-tome — APPROVED** [review]

**Review by alisson-marcelo-azion — APPROVED** [review]

**Pull request merged**